classes/converter_chromium.php

```diff
@@ -107,7 +112,7 @@ protected function generate_pdf_content(moodle_url $proxyurl, string $filename =
                 $page->setUserAgent($options['userAgent']);
             }
 
-            $page->navigate($proxyurl->out(false))->waitForNavigation();
+            $page->navigate($proxyurl->out(false))->waitForNavigation(PAGE::NETWORK_IDLE, 60000);
```

Comment:
Let's add a comment to explain why this is here :)

Reply:
I pushed a fixup commit https://github.com/catalyst/moodle-tool_pdfpages/pull/42/commits/32b6cc531b6beb16026e8994a63cebd427b0b338 :)